todo_app/templates/index.html

```diff
@@ -1,18 +1,51 @@
 {% extends "layout.html" %}
-{% block title %}To-Do App{% endblock %}
-
+{% block title %}Trello Board{% endblock %}
 {% block content %}
   <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <h1 class="display-5">To-Do Board</h1>
+    <p class="lead">Cards</p>
   </div>
-
   <div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+      <table style="width:85%">
+        <tr>
+          <th style="text-align:left">Card Name</th>
+          <th style="text-align:left">Last Activity</th>
+          <th style="text-align:left">Card Status</th>
+          <th style="text-align:center">Action</th>
+        </tr>
+        {% for itm in items %}
+          <tr>
+            <td>{{itm.name}}</td>
+            <td>{{itm.dateLastActivity}}</td>
+            {%if itm.idList=="610beb5a99354405b1f4ec6f" %}
```

Comment:
These hardcoded list ids mean your app won't work if we switch those out with environment variables which we definitely want to be able to do! The nice way to approach this is to use our ViewModel class, so for example if we had a class defined something like:
```python 
class ViewModel():
    def __init__(self, items):
        # store the items
    
    @property
    def todo_items(self):
        # sort through the items and return the relevant ones
```

Then we could use it here like:
```
{% for item  in items.todo_items %}
  # display the relevant html section
{% endfor %}
# repeat for other categories
```